src/ui-kit/experimental/hierarchical/hierarchical-tree/hierarchical-tree.component.html

```diff
@@ -10,7 +10,7 @@
     </sam-hierarchical-tree-header>
 
     <sam-hierarchical-tree-grid [configuration]="configuration" [isSingleMode]="isSingleMode" [gridData]="gridResults | async"
-      (rowChanged)="selecteHierarchyLevel.next($event)" (selectResults)="selectResults$.next($event)" (sorted)="sortLevel.next($event)">
+      (rowChanged)="selecteHierarchyLevel.next($event)" (selectResults)="selectResults$.next($event)" (sorted)="sortLevel.next($event)" (scrolled)="scrolled.next($event)">
```

Comment:
@thomastighe Small typo (I think) -- `selecte`

Reply:
Changed this